galleries/examples/color/color_by_yvalue.py

```diff
@@ -30,3 +30,10 @@
 #    in this example:
 #
 #    - `matplotlib.axes.Axes.plot` / `matplotlib.pyplot.plot`
+#
+# .. tags::
+#
+#    styling: color
+#    styling: conditional
+#    plot-type: line
+#    level: beginner
```

Comment:
I believe these all should be comma separated.

Reply:
I believe RST only requires commas if the directive is written inline. In any case, the built docs are rendering the tags, though I'm not sure why the build is reporting as failing still.

Reply:
You might be right; the build wasn't complaining about the tags as it was in the other PR.